addons/Hierarchical_Table_Of_Contents/src/presenter.js

```diff
@@ -7,6 +7,7 @@ function AddonHierarchical_Table_Of_Contents_create() {
     presenter.keyboardControllerObject = null;
 
     presenter.ERROR_MESSAGES = {
+        EXPAND_DEPTH_NOT_NUMERIC: "Depth of expand is not proper",
```

Comment:
Moja sugestia - "Depth of expand is invalid"

Reply:
Komunikat jest skopiowany z innego addonu. Moim zdaniem nie ma sensu go zmieniać. Nic tym nie zyskamy, a tak jest jakaś spójność